pyproject.toml

```diff
@@ -15,14 +15,15 @@ maintainers = [
 classifiers = [
     "Development Status :: 4 - Beta",
     "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9"
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10"
 ]
 requires-python = ">=3.8"
 
 dependencies = [
     "requests",
     "pandas",
-    "dicttoxml",
+    "dicttoxml>=1.7.8,<2.0.0",
```

Comment:
If any of these libraries released a new major version it could break: Not sure why `dicttoxml` should be treated specially. One practice is to state the dependencies in one place, and then for reproducibility pin to particular version numbers in another.